templates/kafka.service.j2

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Kafka Daemon
-After=zookeeper.service
```

Comment:
In some installs zookeeper runs on the same box as kafka. In those cases it is important that zookeeper start first. In cases where zookeeper is not on the same box systemd handles the situation properly and ignores this line. The line should stay for that first case.